tests/bulk_update/utils_test.py

```diff
@@ -0,0 +1,75 @@
+from tests.unit_test_base import UnitTestBase
+from pyproctor import MonkeyPatcher
+from mock import Mock
+from pyshelf.bulk_update.runner import Runner
+import os.path
+from pyshelf.bulk_update.utils import run
+import logging
+from copy import deepcopy
+
+
+class UtilsTest(UnitTestBase):
+    EXPECTED_KYLE = {
+        "chunkSize": 20,
+        "name": "kyle-long",
+        "logLevel": logging.INFO,
+        "elasticSearchConnectionString": "http://localhost:9200/metadata",
+        "accessKey": "KKKKKKKKKKKKKKKKKKKK",
+        "secretKey": "KKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK"
+    }
+
+    EXPECTED_ANDY = {
+        "chunkSize": 20,
+        "name": "andy-gertjejansen",
+        "logLevel": logging.INFO,
+        "elasticSearchConnectionString": "http://localhost:9200/metadata",
+        "accessKey": "AAAAAAAAAAAAAAAAAAAA",
+        "secretKey": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    }
+
+    def setUp(self):
+        super(UtilsTest, self).setUp()
+        self.run_process_mock = Mock()
+        MonkeyPatcher.patch(Runner, "_run_process", self.run_process_mock)
+
+    def execute(self, bucket=None, chunk_size=None, verbose=False):
+        path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../data/test_config.yaml")
+        args = {
+            "--chunk-size": 20,
+            "--bucket": bucket,
+            "<config-path>": path,
+            "--verbose": verbose
+        }
+
+        if chunk_size:
+            args["--chunk-size"] = chunk_size
```

Comment:
This brings up a question for me. Why not just have `chunk_size=20` for the arg default? Then just setting `"--chunk-size": chunk_size`